formula-android/src/main/java/com/instacart/formula/fragment/FragmentFlowStore.kt

```diff
@@ -1,19 +1,24 @@
 package com.instacart.formula.fragment
 
-import com.instacart.formula.integration.BackStackStore
+import com.instacart.formula.Evaluation
+import com.instacart.formula.Formula
+import com.instacart.formula.FormulaContext
+import com.instacart.formula.RxStream
 import com.instacart.formula.integration.Binding
-import com.instacart.formula.integration.FlowStore
 import com.instacart.formula.integration.DisposableScope
 import com.instacart.formula.integration.FragmentBindingBuilder
+import com.instacart.formula.integration.KeyState
+import com.instacart.formula.integration.LifecycleEvent
+import com.instacart.formula.start
+import com.jakewharton.rxrelay2.PublishRelay
 import io.reactivex.Observable
 
 /**
  * A FragmentFlowStore is responsible for managing the state of multiple [FragmentContract] instances.
  */
 class FragmentFlowStore(
-    private val contractStore: BackStackStore<FragmentContract<*>>,
-    private val store: FlowStore<FragmentContract<*>>
-) {
+    private val root: Binding<Unit, FragmentContract<*>>
+) : Formula<Unit, FragmentFlowState, FragmentFlowState> {
```

Comment:
Moving from `Observable` based implementation to `Formula` based one. Also, removing `FlowStore` dependency here.